### DISCUSSION TITLE
Welcome to tangram-es Discussions!

### DISCUSSION THREAD

**Discussion by matteblair** [tangrams/tangram-es#2282] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

Feel free to comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mallibeerala** [comment]
I am trying to render maps on my laptop through tangram-es engine.
I am able to connect map server and getting .mvt files.
![Screenshot from 2025-04-29 16-10-45](https://github.com/user-attachments/assets/81aedddf-0dce-4b7b-b1c6-bfcd64b37ab2)
I am getting debug window in my laptop & how can I render maps on my laptop display.
